package.json

```diff
@@ -31,7 +31,7 @@
     "anchor": "~0.10.5",
     "async": "1.5.0",
     "captains-log": "~0.11.11",
-    "colors": "1.1.2",
+    "chalk": "^1.1.3",
```

Comment:
(I'm going to pin this to 1.1.3, just to avoid any crazy issues that might come up)